client/app/lib/styl/require-styles.coffee

```diff
@@ -14,4 +14,5 @@ require 'ide/styl'
 # require 'pricing/styl'
 require 'stack-editor/styl'
 require 'new-stack-editor/styl'
+require 'new-stack-editor/views/flexsplit/styl'
```

Comment:
This file only has requires to other app's top level `styl/index.coffee`.
let's move this require to `new-stack-editor/lib/styl/index.coffee`